src/screens/d-apps/others/others.styles.ts

```diff
@@ -1,8 +1,12 @@
+import { Dimensions } from 'react-native';
+
 import { black } from '../../../config/styles';
 import { createUseStyles } from '../../../styles/create-use-styles';
 import { formatSize } from '../../../styles/format-size';
 import { generateShadow } from '../../../styles/generate-shadow';
 
+const width = Dimensions.get('window').width - 40;
```

Comment:
some kind of black magic, maybe its better to use two columns of fixed `flex: 1` + `margin` for content of dapps?

Reply:
also there are `<FlatList />` from RN, that can use `numColumns`